### ISSUE TITLE
PHP warnings

### ISSUE THREAD

**Issue by maikschneider** [maikschneider/bw_captcha#56]
## Setup
* TYPO3 v11.5.33
* php 8.1.27
* apache webserver

## Message
```
Core: Error handler (FE): PHP Warning: Trying to access array offset on value of type null in /var/www/html/...yxz../vendor/blueways/bw-captcha/Classes/Middleware/Audio.php line 53
Core: Error handler (FE): PHP Warning: Undefined array key "plugin." in /var/www/html/...yxz.../releases/567/vendor/blueways/bw-captcha/Classes/Middleware/Audio.php line 53
```

**Comment by maikschneider** [comment]
Should be fixed in #60, released in [v4.0.3](https://github.com/maikschneider/bw_captcha/releases/tag/v4.0.3)

**Comment by tgaertner** [comment]
Tests v4.0.3 - it's fixed now. Thank you!
